frontend/src/components/camp/CampPeriods.vue

```diff
@@ -10,6 +10,7 @@ Displays all periods of a single camp and allows to edit them & create new ones
         <dialog-period-create :camp="camp()">
           <template #activator="{ on }">
             <button-add color="secondary" text
+                        class="my-n2"
```

Comment:
I see a few changes where you tend to make things narrower on this view. I thought it is modern design to have things spaced out a lot, because it works much better for touch devices..?
I don't think it really looks bad with the normal spacing. Quite the opposite, the hover background of this button looks a little clumped up against the separator line below it to me if you add the negative margin...

Reply:
Okay, the "problem" was that "Einstellungen" and "Block-Kategorien" were not on the same line, which I found distracting. Do you have a better idea?

<img width="1012" alt="Screenshot 2021-04-17 at 15 15 13" src="https://user-images.githubusercontent.com/3001985/115114445-b6be5a00-9f8f-11eb-92a5-303ba6a38570.png">